processor/src/main/java/org/mapstruct/ap/internal/model/common/TypeFactory.java

```diff
@@ -565,4 +569,31 @@ static String trimSimpleClassName(String className) {
         return trimmedClassName;
     }
 
+    private boolean isCleared(TypeMirror type) {
```

Comment:
I think the name isCleared has a double connotation, right? IIUYC it means: "safe to use" or "ready to be used". There's also a connotation: erased.